btr-api/src/btr_api/models/submission.py

```diff
@@ -111,15 +111,12 @@ def to_str(submission: Submission):
 
     @staticmethod
     def to_dict(submission: Submission) -> dict:
-        """Return the submission class as a dict."""
+        """Return the submission class as a dict for response purposes."""
 
         return {
```

Comment:
updated to have consistent camel case in response. I also removed the fields that exist inside the payload / are not necessary for us to have in the response (effective_date, business_identifier, submitted_payload).

NOTE: UI only uses the 'payload' and 'id' so this shouldn't effect anything

Reply:
Does this affect the put call I believe I used to_dict there for processing

Reply:
it is used there, but it also only uses the 'payload' from the 'to_dict' response